### PR TITLE
added missing opencv dep in package.xml

### PR DESCRIPTION
**Pull request by JStech**
This was causing the buildfarm to fail.

### CODE REVIEW

**Review by rhaschke — COMMENTED** [review]
Don't you need an `exec_depend` on `libopencv` as well?

**Comment by JStech** [comment]
> Don't you need an `exec_depend` on `libopencv` as well?

Yeah, probably, although I admit I don't really understand what's going on here. Without this change, everything worked fine on my machine, in CI, and in the pre-release test, but then it didn't build on the buildfarm. I was using `moveit_ros/perception` as my reference and it doesn't have any OpenCV dependencies in its `package.xml`. This change just came from asking for help internally here at PickNik.

**Comment by JStech** [comment]
OK, I did some more poking around major packages that use OpenCV, and it looks like having build and exec depends on libopencv-dev is the typical practice.

**Comment by rhaschke** [comment]
Could you provide a link to the failing build farm job please?

**Comment by JStech** [comment]
https://build.ros.org/view/Nbin_uF64/job/Nbin_uF64__moveit_calibration_plugins__ubuntu_focal_amd64__binary/

File: moveit_calibration_plugins/package.xml [hunk 1]
```diff
@@ -30,6 +30,7 @@
   <exec_depend>handeye</exec_depend>
   <exec_depend>criutils</exec_depend>
   <exec_depend>baldor</exec_depend>
+  <exec_depend>libopencv-dev</exec_depend>
```
Comment (rhaschke) [inline moveit_calibration_plugins/package.xml:33]: If you have both, build_depend and exec_depend on the very same package, you can combine them into:
```xml
<depend>libopencv-dev</depend>
```

**Comment by rhaschke** [comment]
Our CI pulls in `opencv` via the MoveIt dependencies (in our docker image). However, it's strange that the prerelease test passed.
Do you have a link to that one as well (in case it was run here via GHA)?

**Review by rhaschke — APPROVED** [review]

**Comment by JStech** [comment]
I don't have a link to the prerelease test--I ran it locally.

File: moveit_calibration_plugins/package.xml (hunk 1 above)
Comment (JStech) [reply moveit_calibration_plugins/package.xml]: :+1:

**Comment by rhaschke** [comment]
Please consider our [merge policies](https://moveit.ros.org/documentation/contributing/maintainer_pr/): PRs like this with several fixup commits should be squash-merged as a single commit. Otherwise the history becomes rather convoluted:
image

**Comment by rhaschke** [comment]
I found the explanation for the succeeding prerelease test: The prerelease test operates on the full repository (while the buildfarm operates on individual package folders only). Thus, because `moveit_calibration_gui` depends on `cv_bridge`, the dependency was pulled in.

**Pull request merged**